src/richie/apps/courses/models/course.py

```diff
@@ -613,6 +620,19 @@ def __str__(self):
         )
 
 
+class LicenceTranslation(TranslatedFieldsModel):
+    master = models.ForeignKey(
+        Licence, models.SET_NULL, related_name="translations", null=True
```

Comment:
Is this recommended by Parler ?

I would have made this field required and deleted this record if the mater object is being deleted. WDYT?

Reply:
Parler does not recommend anything here, in fact they omit this mandatory argument in their own docs.

What you're suggesting is the `models.CASCADE` strategy right? https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.CASCADE

Reply:
yep

Reply:
That’s what I did